vulnerabilities/helpers.py

```diff
@@ -164,3 +165,34 @@ def __lt__(self, other):
         )
 
     return affected_package_with_patched_package_objects
+
+
+def split_markdown_front_matter(lines: str) -> Tuple[str, str]:
+    r"""
```

Comment:
Why a raw string? 
```suggestion
    """
```

Reply:
The doctest contains `\n` and for the sake of clarity I chose raw strings than escaping those.

Reply:
To avoid future question, it would be great to keep this comment `The doctest contains \n and for the sake of clarity I chose raw strings than escaping those.` below the docstring.

Reply:
done